libr/bin/format/mach0/mach0.c

```diff
@@ -1591,7 +1590,7 @@ static bool reconstruct_chained_fixup(struct MACH0_(obj_t) *bin) {
 						bin->chained_starts[seg_idx] = cur_seg;
 						cur_seg->pointer_format = DYLD_CHAINED_PTR_ARM64E;
 						cur_seg->page_size = 0x1000;
-						cur_seg->page_count = (bin->segs[seg_idx].vmsize + (0x1000 - 1)) & ~(0x1000 - 1);
+						cur_seg->page_count = ((bin->segs[seg_idx].vmsize + (0x1000 - 1)) & ~(0x1000 - 1)) / 0x1000;
```

Comment:
maybe better to reuse the pagesize instead of hardcoding the 4KB (0x1000) 3 times. you can defne a variable above like this:

`const size_t ps = 0x1000;` and use `ps` (from `pagesize`) (just trying to make it shorter)